lib/etsy/listing.rb

```diff
@@ -32,7 +32,7 @@ class Listing
 
     include Etsy::Model
 
-    STATES = %w(active removed sold_out expired alchemy)
+    STATES = %w(active removed sold_out expired alchemy edit create private unavailable)
```

Comment:
https://www.etsy.com/developers/documentation/reference/listing#section_listing_states also has a `draft` state. Want to add it?